index.js

```diff
@@ -373,20 +342,26 @@ function continuation (scanner) {
 }
 
 /*
- * <separator>    ::= <summary-sep> | ' #'
+ * <separator>    ::= ": " | " #"
  */
 function separator (scanner) {
+  const start = scanner.position()
   const node = {
     type: 'separator',
     value: ''
   }
-  // <summary-sep>
-  const start = scanner.position()
-  const sum = summarySep(scanner)
-  if (sum instanceof Error) {
-    scanner.rewind(start)
-  } else {
-    return sum
+
+  // ': '
```

Comment:
I like that you've pulled in the space to the `: ` token, one thought though, perhaps make it an optional space, so that people can write:

```
feat:my awesome commit
```

☝️  my thinking is we should make it as hard to screw up the grammar as possible.

Reply:
Yeah, that sounds like a good plan. If it's optional, we might want to consider adding a `whitespace` node. From the linter use case, this could work something like this for a `require-separator-whitespace`-rule:

- `summary` has `separator` node?
- `separator` has `whitespace` sibling?
- `whitespace` value is required length?

Any thoughts on that?

Reply:
What's your reasoning on not wanting to ignore whitespace? Perhaps there are linting rules that still ask for a space after :?